PyInstaller/utils/run_tests.py

```diff
@@ -0,0 +1,36 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2019, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+import os
+import sys
+
+import pytest
+import pkg_resources
+
+
+# Returns a list of PyInstaller tests.
+def PyInstaller_tests():
+    return [os.path.abspath(os.path.join(os.path.dirname(__file__), '../../tests'))]
+
+
+# Return a list of all paths to PyInstller-testable packages.
+def paths_to_test():
+    test_path_list = []
+    for entry_point in pkg_resources.iter_entry_points('pyinstaller40', 'test'):
+        test_path_list += list(entry_point.load()())
+    return test_path_list
+
+
+# Run pytest.
+def run_pytest(*args, **kwargs):
+    pytest.main([*paths_to_test(), *args], **kwargs)
```

Comment:
Assume I'm developing a hook for `scipy`, which requires `numpy` (not sure it it does, just let's assume it). Then this will find all `scipy` and `numpy` tests. But I only want to run the these in `scipy/tests/hooks/part3/`. How can I do this?

(This will always run *all* tests in all paths. User are not able to restrict this to a given directory any more.)

Reply:
```suggestion
    return pytest.main([*paths_to_test(), *args], **kwargs)
```

Reply:
Users could pass additional parameters of `["-k", "<specific tests to run>"]`. See https://docs.pytest.org/en/latest/usage.html#calling-pytest-from-python-code.

Reply:
I think I understand now -- I added a way to select only tests from a given module/submodule.